requirements.txt

```diff
@@ -55,6 +55,7 @@ python-geoip-geolite2==2015.303
 chromelogger==0.4.3
 pypd==1.1.0
 disposable-email-domains
+urllib3==1.24.1
```

Comment:
Why this is needed?

Reply:
This one is needed for ignoring any SSL certificate issue.

Reply:
Can you provide more details on what do you mean?

Reply:
I will be testing this and will update the thread accordingly.